examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -30,12 +35,20 @@ global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
             }
 
```

Comment:
How many times you call shippingOptionsAndRatesFromExternalService()? once or in a loop?

Reply:
@wei-liu-salesforce once